.github/workflows/build-binaries.yml

```diff
@@ -16,18 +19,17 @@ jobs:
             pack-command: :macos
             additional-install: echo "no additional install"
             for: 'macos install kit'
-          - os: ubuntu-latest
+            package-targets: 'darwin-x64'
+          - os: windows-latest
             pack-command: :win
-            additional-install: sudo apt-get install nsis p7zip-full -y
+            additional-install: echo "no additional install"
```

Comment:
I think all the `additional-install` in the matrix could now be removed: it was needed when trying to pack:win on linux, makensis + 7zip were installed.

Now it does not seem like it's of any use.